### PR TITLE
Added the unknown functions within ApplyTrapEffect and other trap information

### PR DESCRIPTION
**Pull request by Jawshoeuh**

### CODE REVIEW

File: symbols/overlay10.yml [hunk 1]
```diff
@@ -1054,6 +1054,12 @@ overlay10:
         EU: 0x4
         NA: 0x4
       description: "The multiplier for damage from the Air Blade (1.5), as a binary fixed-point number (8 fraction bits)"
+    - name: KECLEON_SHOP_BOOST_CHANCE_MULTIPLIER
+      address:
+        NA: 0x22C484C
+      length:
+        NA: 0x4
+      description: "The boosed kecleon shop spawn chance multiplier (~1.2) as a binary fixed-point number (8 fraction bits)."
```
Comment (UsernameFodder) [inline symbols/overlay10.yml:1062]: boosted

Also, apparently the formatter wants the quotes to be removed.

File: symbols/overlay29.yml [hunk 2]
```diff
@@ -1192,6 +1192,107 @@ overlay29:
...
+      address:
+        NA: 0x22EEB18
+      description: |-
+        Turns item in the same room as the tile at the position (usually just the entities's
+        position) into monsters. If the position is in a hallway, convert items in a 3x3 area
+        centered on the position into monsters.
+        
+        r0: entity pointer
+        r1: position
+    - name: ApplyTripTrapEffect
```
Comment (UsernameFodder) [inline symbols/overlay29.yml:1259]: This one is missing a signature in the headers

**Review by UsernameFodder — CHANGES_REQUESTED** [review]

**Comment by UsernameFodder** [comment]
Btw at this point your `master` branch is 30 commits ahead of upstream. Since you're merging upstream/master into your own branch, you can see merge commits in your branch's history, which causes it to look different from upstream, and so each PR is showing all 30 commits from your branch.

I usually solve this by rebasing or fast-forward merging against upstream rather than creating merge commits, but that's maybe harder if you aren't familiar with git. In the future, I would recommend resetting your master branch to exactly match upstream, then creating a separate branch for each new pull request. That way, the master branch can remain in sync with upstream, and you can start fresh on new branches with each new pull request. Let me know if you need help with this.

File: symbols/overlay29.yml [hunk 3]
```diff
@@ -1192,6 +1192,107 @@ overlay29:
...
+        r0: Triggered trap
+        r1: User
+        r2: Target, normally same as user
+        r3: Tile that contains the trap
+        stack[0]: position
+    - name: ApplyGrudgeTrapEffect
+      address:
+        NA: 0x22EF074
+      description: |-
+        Spawns several monsters around the position and gives them all the grudge status
```
Comment (End45) [inline symbols/overlay29.yml:1291]: Doesn't the trap give the grudge status to all the enemies on the floor?

**Review by UsernameFodder — APPROVED** [review]
LGTM

**Pull request merged**